src/components/interaction-designer/block-types/MobilePrimitives_SelectOneResponseBlock.vue

```diff
@@ -106,6 +121,10 @@ export class MobilePrimitives_SelectOneResponseBlock extends mixins(Lang) {
 
   @Prop() readonly flow!: IFlow
 
+  SupportedContentType = SupportedContentType
```

Comment:
Oh, good Q. I've reassigned to make them accessible via `this. SupportedContentType` because `this` is the context provided to the <template> when rendering. This gives access to SupportedContentType within template!